package/boot/uboot-sunxi/patches/003-add-theobroma-a31-pangolin.patch

```diff
@@ -371,5 +371,5 @@
 +
```

Comment:
It looks like `SUNXI_PANGOLIN` is not used at all. I think you can remove the Kconfig file change.
Maybe do it in a separate commit before the u-boot update.

Reply:
I'll drop the Pangolin support completely, it's not even in the target's image Makefile currently - plan to re-add later, but let's drop it for now.